src/hotkey.rs

```diff
@@ -57,12 +57,10 @@ fn compile_modifiers(activators: &Vec<String>) -> u32 {
     let mut code: u32 = 0;
     for key in activators.iter() {
         match key.as_str() {
-            "ALT"   => code = code | MOD_ALT as u32,
-            "CTRL"  => code = code | MOD_CONTROL as u32,
-            "SHIFT" => code = code | MOD_SHIFT as u32,
+            "WIN" => code |= MOD_WIN as u32,
```

Comment:
is adding the WIN key as a modifier okay? According to the docs

>  These keys are labeled with the Windows logo. Keyboard shortcuts that involve the WINDOWS key are reserved for use by the operating system.

Reply:
There is nothing preventing you from using it. I tested setting the hotkey with it and it works. If the user really wants to bind to WIN, it should be their decision to do so.

Reply:
In that case, the max combo length should increase to 5 (all 4 modifiers + 1 virtual key). So the code checking for the hotkey length should be changed likewise. Right now, it will `report_and_exit` if the hotkey length exceeds 4.

Reply:
Ahh, good call! I'll fix that. Doubt anyone will ever configure it that way, as they will need to perform some serious finger gymnastics :P But good to cover all bases